signal/signals.go

```diff
@@ -84,7 +86,7 @@ func ResetDefaultNodeNotificationHandler() {
 
 // TriggerDefaultNodeNotificationHandler triggers default notification handler (helpful in tests)
 func TriggerDefaultNodeNotificationHandler(jsonEvent string) {
-	logger.Trace("Notification received", "event", jsonEvent)
+	logger.Debug("Notification received", zap.String("event", jsonEvent))
```

Comment:
I've seen on web that there're hacks to add `TRACE` level to zap. Not sure if we want it, but it makes sense to have I think.

Reply:
Let me keep it as debug now. I don't think it will make much sense for `TRACE` once categorized logging  is implemented.

Reply:
Sure, I didn't mean to do it now